libcaf_core/caf/actor_registry.hpp

```diff
@@ -51,9 +52,21 @@ class actor_registry {
   /// Returns the local actor associated to `key`.
   strong_actor_ptr get(actor_id key) const;
 
+  /// Returns the local actor associated to `key`.
+  template<typename T>
```

Comment:
please change to `template <class T>`, also for the others. :)

Reply:
Ok, fixed!